source/quick-start.txt

```diff
@@ -50,6 +50,13 @@ If you are using `Gradle <https://gradle.org/>`__, add the following to your
 
 .. include:: /includes/quick-start/gradle-versioned.rst
 
+.. note::
+   
+   We recommend that you use a package manager to install the
+   {+driver-short+}. However, if you must download the driver
+   and dependency packages, you can find the JAR files in the `Maven
+   repository <https://mvnrepository.com/artifact/org.mongodb/mongodb-driver-sync/>`__.
```

Comment:
Issue:

The current link only contains the mongodb-driver-sync JAR and not any of the dependencies. 

Suggestion:
I think it could be more helpful to specify the group (which references all org.mongodb JARs) and which ones are necessary.

```suggestion
   and its dependencies, you can find the JAR files in the `MongoDB Group Maven
   repository <https://mvnrepository.com/artifact/org.mongodb/>`__. The Java Sync
   driver requires the ``mongodb-driver-core``, ``bson``, and ``mongodb-driver-sync``
   JAR files.
```